.github/CODEOWNERS.md

```diff
@@ -0,0 +1 @@
+* @Valeryibarra @renardete @leonleo997 @holgiosalos
```

Comment:
change this to be `* @germandavid85 @aperdomob @holgiosalos @danielgalvis98 @veronicatofino`

Reply:
done